.travis.yml

```diff
@@ -11,19 +6,27 @@ env:
 matrix:
   include:
     - os: osx
+      osx_image: xcode9.0
       language: node_js
       node_js: "8"
     - os: linux
-      language: generic
+      services: docker
+      # even though we don't build anything in the "docker" mode, it's best to define the language as node_js
+      # so that we can manipulate node_modules if needed and keep things consistent
+      language: node_js
+      node_js: 8
 
 cache:
   directories:
   - node_modules
   - $HOME/.cache/electron
   - $HOME/.cache/electron-builder
-
+  - $HOME/.npm
+  - $HOME/.yarn-cache
+    
 before_install:
-  - mkdir -p /tmp/git-lfs && curl -L https://github.com/github/git-lfs/releases/download/v2.3.0/git-lfs-$([ "$TRAVIS_OS_NAME" == "linux" ] && echo "linux" || echo "darwin")-amd64-2.3.0.tar.gz | tar -xz -C /tmp/git-lfs --strip-components 1 && /tmp/git-lfs/git-lfs pull
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install git-lfs; fi
```

Comment:
using brew is not performant (update will be called), please use old approach.

Reply:
Ok